engine/src/efx.h

```diff
@@ -64,9 +64,11 @@ class Fixture;
 #define KXMLQLCEFXDiamondAlgorithmName      QString("Diamond")
 #define KXMLQLCEFXSquareAlgorithmName       QString("Square")
 #define KXMLQLCEFXSquareChoppyAlgorithmName QString("SquareChoppy")
+#define KXMLQLCEFXSquareTrueAlgorithmName  QString("SquareTrue")
```

Comment:
Please fix indentation

Reply:
Fixed in 5a3d1edc46f3b7e7601b6d34ee5964406f30adc4  (sorry!)